alpha/alpha-core/src/main/java/org/apache/servicecomb/saga/alpha/core/EventScanner.java

```diff
@@ -92,11 +96,11 @@ private void updateCompensatedCommands() {
         });
   }
 
-  private void deleteDuplicateSagaEndedEvents() {
+  private void deleteDuplicateEvents() {
     try {
-      eventRepository.deleteDuplicateEvents(SagaEndedEvent.name());
+      eventRepository.deleteDuplicateEvents(Arrays.asList(TxAbortedEvent.name(), SagaEndedEvent.name()));
```

Comment:
we don't delete duplicate events other than SagaEndedEvent. we need to keep track of all events for troubleshooting and auditing purpose.